doc/tutorials/feature_interaction_constraint.rst

```diff
@@ -127,11 +127,19 @@ first and second constraints (``[0, 1]``, ``[2, 3, 4]``).
   Source(source, format='png').render('../_static/feature_interaction_illustration3', view=False)
   Source(source, format='svg').render('../_static/feature_interaction_illustration3', view=False)
 
+.. list-table::
```

Comment:
Out of curiosity, is there a way to align two figures in the same line (left and right)?

Reply:
Some googling gave me https://github.com/sphinx-doc/sphinx/issues/4003 and https://stackoverflow.com/questions/10219634/image-grid-in-restructuredtext-sphinx. So I went for the first one.

Is there a way to see the rendered documentation from the CI?

Reply:
> Is there a way to see the rendered documentation from the CI?

Unfortunately no.  The doc is only deployed when it's merged into the master branch.  But to have a quick view, you can install the needed dependencies specifed by `doc/xgboost_doc.yml` and call `make html` under the doc subdirectory.

Reply:
In https://github.com/dmlc/xgboost/pull/7219/commits/ef751d7280920dc967b5360c852c0572d4f3c52e I put the figures as images in a grid table. This is the best, I could get working. But I'm not an expert for rst and sphinx.